stix2/test/test_observed_data.py

```diff
@@ -761,14 +761,13 @@ def test_file_example_with_WindowsPEBinaryExt():
 
 
 def test_file_example_encryption_error():
-    with pytest.raises(stix2.exceptions.DependentPropertiesError) as excinfo:
-        stix2.File(name="qwerty.dll",
-                   is_encrypted=False,
-                   encryption_algorithm="AES128-CBC"
-                   )
+    file_ = stix2.File(name="qwerty.dll", is_encrypted=False,
```

Comment:
Why is the result of this test changing? As far as I can tell, it should be invalid to create a File with an `encryption_algorithm` when `is_encrypted=False`. If the changes to `getattr` caused this constraint to stop working, we should fix that.